### DISCUSSION TITLE
Update to Russian locale [ru]

### DISCUSSION THREAD

**Discussion by Communist02** [aclap-dev/video-downloadhelper#859] [Language & Translation]
[ru.txt](https://github.com/user-attachments/files/15786347/ru.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
